docs/cart-and-checkout-blocks/hooks/hook-alternatives.md

```diff
@@ -0,0 +1,456 @@
+| `woocommerce_get_cart_page_permalink`                 | ❌                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_get_cart_url`                            | ❌                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_checkout_registration_enabled`           | ✅                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_get_checkout_page_permalink`             | ❌                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_get_checkout_url`                        | ❌                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_checkout_get_value`                      | ❌                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_default_address_fields`                  | ❌                | This does not seem to have any effect                                                                                                                        |
+| `default_checkout_billing_country`                    | ❌                | This does not seem to have any effect                                                                                                                        |
+| `default_checkout_shiping_country`                    | ❌                | This does not seem to have any effect                                                                                                                        |
+| `woocommerce_get_country_locale`                      | ✅                |                                                                                                                                                              |
+| `woocommerce_get_country_locale_default`              | ❓                |                                                                                                                                                              |
+| `woocommerce_get_country_locale_base`                 | ❓                |                                                                                                                                                              |
```

Comment:
I'm not sure at what point should we start to validate? I'd also like to validate the `X` ones as plenty of them actually work fine, like those 2.

Reply:
I think we can start validating from now, like it could be a cycle focus to check these hooks. I think it would be a significant chunk of time to do this though which is why I don't want to spend time during this cycle re-validating. cc @ralucaStan